front/src/modules/simulationResult/components/SpaceTimeChartWrapper/SpaceTimeChartWrapper.tsx

```diff
@@ -139,9 +146,29 @@ const SpaceTimeChartWrapper = ({
   const [draggingState, setDraggingState] = useState<DraggingState>();
 
   const isTimetableItemValid = useMemo(() => {
+    let selectedItemId: TrainScheduleId | PacedTrainId | OccurrenceId = selectedProjectionId;
+
+    if (isOccurrenceId(selectedProjectionId)) {
+      selectedItemId = extractPacedTrainIdFromOccurrenceId(selectedProjectionId);
+    }
+
     const timetableItemUsedForProjectionWithDetails = timetableItemsWithDetails?.find(
-      (item) => item.id === selectedProjectionId
+      (item) => item.id === selectedItemId
     );
+
+    if (
+      timetableItemUsedForProjectionWithDetails &&
+      'exceptions' in timetableItemUsedForProjectionWithDetails &&
+      isOccurrenceId(selectedProjectionId)
+    ) {
+      const exeptionsUsedForProjection = timetableItemUsedForProjectionWithDetails.exceptions.find(
+        (exception) =>
+          exception.occurrence_index ===
+          extractOccurrenceIndexFromOccurrenceId(selectedProjectionId)
+      );
+      if (exeptionsUsedForProjection) return exeptionsUsedForProjection?.summary?.isValid ?? false;
```

Comment:
Can you test if exception.summary is not undefined ? Otherwise, it means that the summary has not been loaded (which should not happen at this stage, but we can't be sure).

```suggestion
      if (exception?.summary) return exception.summary.isValid;
```

Reply:
Done